templates/patterns/fact_figures/fact_figures.ui_patterns.yml

```
@@ -0,0 +1,99 @@
+fact_figures:
+  label: "Fact and Figures"
+  description: "Fact and Figures pattern used for the paragraph of same type."
+  settings:
+    style:
+      type: select
+      label: Style
+      description: "Style of the composition"
+      options:
+        default: default
+        block: block
+      preview: default
+    display_icons:
+      type: boolean
+      label: "icons visibility"
+      description: "Toggle icon visibility"
+      preview: true
+    responsive_columns:
+      type: number
+      label: "Responsive Columns"
+      description: "One, two or three columns layout for responsive"
+      preview: 3
+    grid_columns:
+      type: number
+      label: "Grid Columns"
+      description: "One, two or three columns layout for grid"
+      preview: 3
+    gutter:
+      type: number
+      label: "Gutter"
+      description: "Gutters between elements."
+      preview: 4
+    equal_height:
+      type: boolean
+      label: "Equal height"
+      description: "Equal height between each item."
+      preview: true
+  fields:
+    title:
+      type: "text"
+      label: "Title"
+      description: "Title of the block"
+      preview: "Fact and figures block"
+    items:
+      type: "array"
+      label: "List items"
+      description: "List of items"
+      preview:
+        - icon:
+            name: "x-diamond-fill"
+            size: "l"
```

we should send only the icon name
```suggestion
            - icon: "x-diamond-fill"
```